<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServeurCrawler</class>
 <widget class="QWidget" name="ServeurCrawler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>329</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Serveur chasse au trésor</string>
  </property>
  <widget class="QPushButton" name="pushButtonLancementServeur">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Lancement</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuitter">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPortEcoute">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1024</number>
   </property>
   <property name="maximum">
    <number>65536</number>
   </property>
   <property name="value">
    <number>8888</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelPortEcoute">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Port d'écoute</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>ServeurCrawler</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
